Branch-SDK/src/main/java/io/branch/referral/BranchError.java

```diff
@@ -143,6 +145,9 @@ private String initErrorCodeAndGetLocalisedMessage(int statusCode) {
         } else if (statusCode >= 400 || statusCode == ERR_BRANCH_INVALID_REQUEST) {
             errorCode_ = ERR_BRANCH_INVALID_REQUEST;
             errMsg = " The request was invalid.";
+        } else if (statusCode == ERR_IMPROPER_REINITIALIZATION) {
```

Comment:
Question about this new error message. Were users seeing an error, but with an inappropriate message?

Reply:
No their (reinitialization) callback was simply never invoked.